ch05construction/02conventions.ipynb

```diff
@@ -170,7 +177,9 @@
     "* Brace style\n",
     "* Line length\n",
     "* Indentation\n",
-    "* Whitespace/Tabs\n"
+    "* Whitespace/Tabs\n",
+    "\n",
+    "Inconsistence will produce a mess in your code! Some would make your code harder to read, whereas others choices may affect the code. For example, if you copy/paste code with tabs in a place that's using spaces, they may appear OK in your screen but it will fail when running it."
```

Comment:
```suggestion
    "Inconsistency will produce a mess in your code! Some choices will make your code harder to read, whereas others may affect the code. For example, if you copy/paste code with tabs in a place that's using spaces, they may appear OK in your screen but it will fail when running it."
```